docs/pages/web/searchguide.tsx

```diff
@@ -0,0 +1,363 @@
+- \`accessibilityExpanded\`: informs the screen reader whether the searchguide-anchored Popover-based component is currently open or closed. It populates [aria-expanded](https://www.w3.org/TR/wai-aria-practices/examples/menu-searchguide/menu-searchguide-links.html).
+
+Also consider:
+- Ensure that each SearchGuide uses the correct markup for lists. Typically screen readers announce the number of items in a list and announce each item.
+- Limit the number of SearchGuides to 9 per search query. Horizontal scrolling can be challenging to users with mobility issues, or users of assistive devices such as screen readers.
+`}
+          title="ARIA attributes"
+        />
+      </AccessibilitySection>
+
+      <LocalizationSection
+        name={generatedDocGen?.displayName}
+        noDefaultLabelProvider
+        notes="Avoid truncating SearchGuide text whenever possible. Refer to the [SearchGuide usage guidelines](#Usage-guidelines) for more information"
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`Use the \`color\` prop to change the background color of the SearchGuide. The color prop can be one of the following values:`}
+          title="Colors"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample code={colors} layout="column" name="Colors" previewHeight={500} />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          description={`
+            Use the \`thumbnail\` prop to display an image to the left of the text. The \`thumbnail\` prop can be an [Avatar](/web/avatar), [AvatarGroup](/web/avatargroup), [Image](/web/image), or [Icon](/web/icon).
+  `}
+          title="Thumbnail"
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={
+              <SandpackExample
+                code={avatarExample}
+                name="Searchguide example with avatar."
+                previewHeight={PREVIEW_HEIGHT}
+              />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          columns={2}
+          description={`
+Use the \`selected\` prop to indicate that the SearchGuide is in a selected state.
+`}
+          title="States"
+        >
+          <MainSection.Card
+            cardSize="md"
+            sandpackExample={
+              <SandpackExample
+                code={selectedStateExample}
+                name="Selected state searchguide example."
+                previewHeight={150}
+              />
+            }
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          columns={2}
+          description={`
+When a SearchGuide query is broad and can be further refined (example: Season), an **arrow-down** icon can be added at the end, to trigger a Sheet with a cluster of wrapped SearchGuides.
```

Comment:
Can we just ad the example and no description?


Prone to errors... 1.  (example: Season), but the example has another category
2. trigger a Sheet, we are triggering a Popover...
3. wrapped SearchGuides or SearchLinks...



just dont add a description

Reply:
Done